cmd/libs/go2idl/conversion-gen/generators/conversion.go

```diff
@@ -122,6 +133,16 @@ func existingConversionFunctions(context *generator.Context) conversions {
 	return preexisting
 }
 
+func isAPIPackage(path string) bool {
+	apiPackages := []string{"k8s.io/kubernetes/pkg/api/", "k8s.io/kubernetes/pkg/apis/"}
```

Comment:
Explicit "package comment" might be the best option - will try to look into it tomorrow between meetings.